services/scripts.py

```diff
@@ -1,61 +1,66 @@
 # Music recommendation application
 import csv
 
-from cfg import TableEnum
+from cfg import TableIndexes
 
 
 class Song:
     def __init__(self, row: list[str]) -> None:
-        self.id = int(row[TableEnum.ID.value])
```

Comment:
why..? as far as i know, we did it on purpose.. but i could be wrong

Reply:
Because I parse it in function `__parse_row`.

Reply:
> Because I parse it in function `__parse_row`.

sry, my bad